webui/src/app/components/badges/badges.component.ts

```diff
@@ -11,10 +11,11 @@ import { DataDeploymentService } from '../../services/data-deployment.service';
 import { UiKitPaginate } from '../../models/kit/paginate';
 import { ActivatedRoute } from '@angular/router';
 import { BadgesResolver } from '../../resolver/resolve-badges';
-import { BehaviorSubject } from 'rxjs/BehaviorSubject';
+import { BehaviorSubject } from 'rxjs';
 import { OuiPaginationComponent } from '../../kit/oui-pagination/oui-pagination.component';
 import { BadgeShieldsIOBean } from '../../widget/badgewidget/badgewidget.component';
 import { UIChart } from 'primeng/primeng';
+import { Observable } from 'rxjs/internal/Observable';
```

Comment:
Sounds very weird to rely on someone else' internals! Auto-import typo for `rxjs/Observable`?